debian/mstpd.lintian-overrides

```diff
@@ -0,0 +1 @@
+mstpd: read-in-maintainer-script
```

Comment:
hmmm ; it may be interesting to try to add this to our `.travis.yml`
i'll take a look